CVLByExample/summary/simple/README.md

```diff
@@ -0,0 +1,32 @@
+This directory contains examples for [Function Summaries](https://docs.certora.com/en/latest/docs/cvl/cvl2/changes.html#summaries-only-apply-to-one-contract-by-default:~:text=a%20function%E2%80%99s%20sighash).-,Summaries%20only%20apply%20to%20one%20contract%20by%20default,%EF%83%81,-In%20CVL%201)
+
+The interface `IntGetter` has the functions `get1()` and `get2()` that have no implementation. 
+Function summaries are used instead of their missing implementations. Since IntGetter is an interface and not a contract a 
+wildcard entry is used.
```

Comment:
we use wildcards not only for interface, so this is confusing. wildcard is for multiple possible call targets, maybe the documentation explains this?

Reply:
This explains that for interface you need a wildcard. It doesn't say it is the only case you need it.